api/PclusterApiHandler.py

```diff
@@ -33,7 +33,14 @@
 SECRET_ID = os.getenv("SECRET_ID")
 ENABLE_MFA = os.getenv("ENABLE_MFA")
 SITE_URL = os.getenv("SITE_URL", API_BASE_URL)
+SCOPES_LIST = os.getenv("SCOPES_LIST")
+REGION = os.getenv("AWS_DEFAULT_REGION")
+TOKEN_URL = os.getenv("TOKEN_URL", f"{AUTH_PATH}/oauth2/token")
+AUTH_URL = os.getenv("AUTH_URL", f"{AUTH_PATH}/login")
+JWKS_URL = os.getenv("JWKS_URL")
+AUDIENCE = os.getenv("AUDIENCE")
 USER_ROLES_CLAIM = os.getenv("USER_ROLES_CLAIM", "cognito:groups")
+REDIRECT_URL = f"{SITE_URL}/login"
```

Comment:
suggestion(non-blocking): Shall we document this new stuff? (the README maybe?)

Reply:
I see. Would this affect our ability to run the backend locally though?

Reply:
No, I don't think so, as we already have a runbook with the env variables needed to run locally